locales/en.js

```diff
@@ -410,7 +414,9 @@ const en = {
     gender: 'Gender',
     birthday: 'Birthday',
     tags: 'Tag',
-    createMember: 'Create Member'
+    createMember: 'Create Member',
+    recentOrders: 'Recent Five Orders',
```

Comment:
Change to Recent Orders, just in case  we decide to have more or less than 5 orders.